packages/plugins/Traderxyz/src/SNSAdaptor/PreviewOrderView.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, Chip, Grid, IconButton, Typography, Avatar } from '@mui/material'
+import { ArrowBack } from '@mui/icons-material'
+import type { nftData } from '../types'
+import { useI18N } from '../locales/i18n_generated'
+
+export const PreviewOrderView = (props: {
+    nftList: nftData[] | undefined
+    setDisplaySection: Function
+    classes: Record<string, string>
+    amountLabel: string
+}): JSX.Element => {
+    const t = useI18N()
+    const { nftList, classes, amountLabel } = props
+    const child = (
+        <img
+            alt="no-image"
+            src="https://trader.xyz/images/missing-img-lg.png"
+            className={classes.previewBoxInnerGridContainerItemImg}
+        />
+    )
+
+    const previewImages = nftList?.map((item: nftData) => {
+        return (
+            <>
```

Comment:
The fragment is redundant and lost `key` here.

Reply:
done, added key and remove the `<></>`